### PR TITLE
Fix/menu bar navigation

### PR DESCRIPTION
**Pull request by alexandrehcdc**
This PR contains changes from the integration of the menu bar navigation.

For the moment, it is related to construction site selection screen/others menu. Further changes will be integrated in the near future.

### CODE REVIEW

**Review by rachidcalazans — APPROVED** [review]

File: StantUiIosLibrary/Classes/Cells/InternPersonCell.swift [hunk 1]
```diff
@@ -33,24 +51,107 @@ public class InternPersonCell: UITableViewCell {
...
+            self.mailImage.set(color: .darkGrayStant)
+        }
+        
+        if self.callButton == nil {
+            self.callImage.set(color: .darkGrayStant)
+        }
+    }
+    
+    private func positionOptionalElements() {
+
```
Comment (liuvasconcelos) [inline StantUiIosLibrary/Classes/Cells/InternPersonCell.swift]: Remove extra line

File: StantUiIosLibrary/Classes/Cells/InternPersonCell.swift [hunk 2]
```diff
@@ -33,24 +51,107 @@ public class InternPersonCell: UITableViewCell {
...
+                                                    left: 0,
+                                                    bottom: 18,
+                                                    right: 16),
+                              size: CGSize(width: 20, height: 20))
+
+        if let callButton = self.callButton {
+            self.addSubview(callButton)
+            
+            self.callButton?.anchor(top: self.topAnchor,
+                                    leading: nil,
```
Comment (liuvasconcelos) [inline StantUiIosLibrary/Classes/Cells/InternPersonCell.swift]: Do you know that you dont need to send this nil here?

File: StantUiIosLibraryDemo/StantUiIosLibraryDemoTests/InternCell/InternActionCellTests.swift [hunk 3]
```diff
@@ -24,14 +25,15 @@ class CellWithImageTests: XCTestCase {
...
-
     }
 
     override func tearDown() {
         self.cell            = nil
         self.cellIcon        = nil
         self.imageSize       = nil
+        self.imageColor      = nil
         self.textDescription = nil
+
```
Comment (liuvasconcelos) [inline StantUiIosLibraryDemo/StantUiIosLibraryDemoTests/InternCell/InternActionCellTests.swift]: Remove extra line.

File: StantUiIosLibrary/Classes/Cells/InternPersonCell.swift (hunk 1 above)
Comment (alexandrehcdc) [reply StantUiIosLibrary/Classes/Cells/InternPersonCell.swift]: Done!

File: StantUiIosLibrary/Classes/Cells/InternPersonCell.swift (hunk 2 above)
Comment (alexandrehcdc) [reply StantUiIosLibrary/Classes/Cells/InternPersonCell.swift]: You are right, just fixed it!

**Review by liuvasconcelos — APPROVED** [review]

**Pull request merged**